From 9ae4355cdf7fae14f96fffa9033d073497e5c5b3 Mon Sep 17 00:00:00 2001
From: "Tan, Raymond" <raymond.tan@intel.com>
Date: Fri, 17 Apr 2015 13:12:57 +0800
Subject: [PATCH 236/389] spi-pxa2xx: add support up to 4 slaves per master -
 RTC #87333

This patch is adopted from R1.1 to support up to 4 slaves per master
on Quark X1000 SoC platform.

The original author of the changes in R1.1 is
Josef Ahmad <josef.ahmad@intel.com>.

Signed-off-by: Tan, Raymond <raymond.tan@intel.com>
---
 drivers/spi/spi-pxa2xx-pci.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/spi/spi-pxa2xx-pci.c b/drivers/spi/spi-pxa2xx-pci.c
index 88b8f91..d35925e 100644
--- a/drivers/spi/spi-pxa2xx-pci.c
+++ b/drivers/spi/spi-pxa2xx-pci.c
@@ -96,7 +96,7 @@ static struct pxa_spi_info spi_info_configs[] = {
 	[PORT_QUARK_X1000] = {
 		.type = QUARK_X1000_SSP,
 		.port_id = -1,
-		.num_chipselect = 1,
+		.num_chipselect = 4,
 		.max_clk_rate = 50000000,
 	},
 };
-- 
1.9.3

